### DISCUSSION TITLE
If you want to use `make`, type `mingw32-make` instead.

### DISCUSSION THREAD

**Discussion by Welding-Torch** [Welding-Torch/installc#2] [General]
I am considering adding an alias to Install C so that typing `make` leads you to execute `mingw32-make` instead. Do comment and let me know if you are in _support of_ / _against this_ feature.

Note: Despite it's name, `mingw32-make` can be used to make 64-bit binaries.

**Comment by parlortrickss** [comment]
It would be handy to have that alias. I went to use `make` only to find i had to use `mingw32-make`. Not a huge deal but i was expecting the normal and had to search. I think it would reduce the friction, and also align with your goals of making C easier to install and use.

**Reply by Welding-Torch** [reply]
Thanks for the feedback! I'll implement this alias in the next release.
